### DISCUSSION TITLE
How to make tab bar not transluscent?

### DISCUSSION THREAD

**Discussion by x24sousa** [black7375/Firefox-UI-Fix#829] [Q&A]
So my tab bar, on the unselected tabs, has a translucent look to it that I do not care for:

![image](https://github.com/black7375/Firefox-UI-Fix/assets/140837729/cc3cf10f-4f7a-476b-a662-9a4d0f79caf7)

I would prefer it to look like this, which happens when Firefox is not the active window:

![image](https://github.com/black7375/Firefox-UI-Fix/assets/140837729/a17d14ca-7ac6-4f54-a6a3-a69009195cb0)

I am aware of [this section of the Options](https://github.com/black7375/Firefox-UI-Fix/wiki/Options#theme) page, but was unsure how to actually implement it. I would like my FF to look like the 1st and 3rd examples, according to Light or Dark theme.

![image](https://github.com/black7375/Firefox-UI-Fix/assets/140837729/3d1bc64e-cbd5-4f85-b035-14e6a8163ff3)

Thank you!

**Comment by black7375** [comment]
Okay, I think it would be a good idea to add an diable option.

**Reply by x24sousa** [reply]
Do you mean some sort of toggle, or easily changeable option?

**Reply by black7375** [reply]
You mean you want to remove the background color of the tab bar, right?
I'm busy with other things today, so more will be added sometime next week.
